examples/showcase_1kt.rs

```diff
@@ -0,0 +1,240 @@
+//! An example showing Tock's capabilitities ported to CherryMotes, on 1KT.
+//! Every T seconds (a configurable constant) it infinitely sends a frame
+//! with a tuple (timestamp, CherryMote id, sequence number, measured temperature)
+//! and after each send receives N - 1 (a configurable constant) frames from other CherryMotes
+//! and prints them to console.
+
+#![no_main]
+#![no_std]
+#![forbid(unsafe_code)]
+
+use core::fmt::{Display, Write as _};
+use libtock::alarm::{Alarm, Milliseconds};
+// use libtock::console::Console;
+use libtock::console_lite::ConsoleLite;
+use libtock::ieee802154::{Ieee802154, RxBufferAlternatingOperator, RxOperator as _, RxRingBuffer};
+use libtock::runtime::{set_main, stack_size};
+use libtock::temperature::Temperature;
+
+set_main! {main}
+stack_size! {0xA00}
+
+/// Number of CherryMotes executing this app at the same time.
+const N_MOTES: usize = 2;
+
+/// Interval between frame transmissions.
+const N_SECS: u32 = 3;
+const SLEEP_TIME: Milliseconds = Milliseconds(1000 * N_SECS);
+
+fn get_cherry_id() -> Option<&'static str> {
+    option_env!("HENI_DEVICE")
```

Comment:
Docelowo tu będzie wyliczanie z MAC?

Reply:
Wydaje mi się, że jeśli `None` to panikować, bo bez ID wysyłanie wiadomości itd. nie ma sensu z punktu widzenia całej apki.

Reply:
A, może myślimy o czymś innym: nie mam na myśli ID CherryMote'a (`139`, `656` itd.) – bo mapowanie MAC Supervisora <-> MAC Node'a trzeba stwierdzić ,,empirycznie'' – tylko jakieś ID tego Node'a, które pozwalałoby w obrębie niskomoc`owego eksperymentu rozróżniać mote'y. Osobiście bym się nie męczył, tylko używał w tym celu całego MACa z FCFG/CCFG. Jeśli potrzebujesz czegoś krótszego, to można sufiks tego MACa, bo prefiksy powinny być w 1KT takie same.
Pomysł ze zmienną środowiskową jest fajny, ale obawiam się, że będzie to niepotrzebnie komplikować deployment w skali 1KT.